aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -257,17 +263,35 @@ When using the out-of-process hosting model, the app might not shut down immedia
 ::: moniker-end
 
 ## Start-up error page
-
 ::: moniker range=">= aspnetcore-2.2"
 
-*Only applies to out-of-process hosting.*
+Both inprocess and outofprocess have custom error pages when they fail to start the application.
+
+If the ASP.NET Core module fails to find either the inprocess or out of process request handler, a *500.0 In-Process/Out-Of-Process Handler Load Failure* status code page appears.
+
+![502.5 Process Failure Status Code Page](aspnet-core-module/_static/ANCMV2-500-inproc.png)
+![502.5 Process Failure Status Code Page](aspnet-core-module/_static/ANCMV2-500-outofproc.png)
+
+For inprocess, if the ASP.NET Core module fails to start the application, a *500.30 Start Failure* status code page appears.
```

Comment:
* in-process over "inprocess" ... and perhaps add "hosting" to it
* No comma before a subordinate conjunction (if) ... but we argue this internally. :smile:
* Capitalize the module name
* "app" over "application"
* Dash in the error status code and description

Here's a cut-'n-paste ...

> For in-process hosting if the ASP.NET Core Module fails to start the app, a \*500.30 - Start Failure\* status code page appears.